src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.StateManager.cs

```diff
@@ -122,12 +86,7 @@ private static ImmutableDictionary<DiagnosticAnalyzer, StateSet> CreateStateSetM
                         // #1, all de-duplication should move to DiagnosticAnalyzerInfoCache
```

Comment:
Does this todo still apply?  Or is the de-duplication here fine at this point?

Reply:
will check!

Reply:
4fa21f55b47